src/IRMatch.h

```diff
@@ -1339,6 +1339,10 @@ constexpr bool and_reduce(bool first, Args... rest) {
     return first && and_reduce(rest...);
 }
 
+constexpr int const_min(int a, int b) {
```

Comment:
// TODO: this can be replaced with std::min() once we require C++14 or later

Reply:
Don't we already? I thought we had OK'd C++17 features.

Reply:
Nope, C++11 is all we count on at the moment (though C++14 can be enabled via LLVM settings in some setups, I think).

We had agreed to move to C++17 baseline for Halide 12 (once we cut the Halide 11 release).

Reply:
Ah, I forgot we were waiting till post-release. I told @rootjalex he was good to use constexpr if and was suddenly worried that was a lie. So we can use c++17 in branches that won't merge until after the release.